widget/check_group.go

```diff
@@ -18,10 +20,15 @@ type CheckGroup struct {
 	Options    []string
 	Selected   []string
 
+	Validator           fyne.CountValidator
```

Comment:
I can't help but shake the idea that this count/int validation isn't the primary type for `Check` (and possibly radio as well)... They are `string` or `[]string` data sources...
Will this possibly clash in the future if we want to validate the data more directly?
Should it be `SelectionValidator` instead?

Reply:
I guess you mean `StringValidator`? Hmm, that’s hard. I don’t think verifying strings makes sense for a widget where the user doesn’t input their own fields.

Sure, the developer might have set up some way to popular the items in the Select widget or Radio, but for the most part, it is just a case of users selecting from a predefined list. I personally think the the number of selections make more sense.

Reply:
Yes good points. But by calling this `Validator` we are committing to it being the main way of validating - but I am not sure that this makes sense, which is why I was suggesting the `SelectionValidation` name instead. When I look at other usage of `Validator` it is more comparing to the data itself rather than quantity of data selected...

Reply:
Ah, I see now. I don't think I quite understood what you meant before. Given that the new validator was called `SelectionValidator` before, it caused some confusion.

Reply:
> Ah, I see now. I don't think I quite understood what you meant before. Given that the new validator was called `SelectionValidator` before, it caused some confusion.

Yeah sorry about that, I could have been clearer.